test/test_nn.py

```diff
@@ -4305,6 +4305,51 @@ def test_conv_double_backward_cuda(self):
                                 "\ninp_size: " + str(inp_size) +
                                 "\ndilation: " + str(dilation))
 
+    def run_grad_conv_test(self, func_forward, func_backward, dim=1, gradient='input'):
+        for kern, inp_size in [(3, 6), (3, 7), (4, 9)]:
+            for batch, stride, padding, chan_in, chan_out, dilation in \
+                    product([1, 2], [1, 2], [0, 1, 2], [2], [3], [1]):
+
+                input_shape = [batch, chan_in]
+                weight_shape = [chan_out, chan_in]
+                for _ in range(dim):
+                    input_shape.append(inp_size)
+                    weight_shape.append(kern)
+
+                input = torch.autograd.Variable(torch.randn(input_shape), requires_grad=True)
+                weight = torch.autograd.Variable(torch.randn(weight_shape), requires_grad=True)
```

Comment:
`Variable` is deprecated. You can pass `requires_grad=True` directly to `.randn`